bundle-chroot-builder.py

```diff
@@ -271,6 +271,33 @@ def progress(r):
     for r in results_list:
         r.get()
 
+    """Read the URL values from builder.conf and insert them into os-core-update to swupd knows where to pull content from"""
+    buildconf='/usr/share/defaults/bundle-chroot-builder/builder.conf'
+    if os.path.isfile('/etc/bundle-chroot-builder/builder.conf'):
+        buildconf = '/etc/bundle-chroot-builder/builder.conf'
+    if args.config:
+        buildconf = args.config
+    config = configparser.ConfigParser()
+    config.read(buildconf)
+
+    """Read the configuration file for our script values"""
+    conf = config['swupd']
+    bundlename = conf['BUNDLE']
+    url = conf['URL']
+    contenturl = conf['CONTENTURL']
+    versionurl = conf['VERSIONURL']
+    formatname = conf['FORMAT']
+
+    os.mkdir(out_dir + bundlename + "/usr/share/defaults/swupd/")
+    with open(out_dir + bundlename + "/usr/share/defaults/swupd/url", "w") as file:
```

Comment:
Probably also just create a temp var of the join()'d path since you use it a bit.